src/gateways/ElasticsearchGateway.test.ts

```diff
@@ -10,8 +10,30 @@ describe('ElasticsearchGateway', () => {
   let gateway: ElasticsearchGateway;
 
   beforeEach(() => {
+    const elasticSearchResponse = {
+      body: {
+        hits: {
+          hits: [
```

Comment:
I can believe this _is_ how Elasticsearch works given previous experiences of it, but is the response really a `hits` object with an array of `hits` inside?

Reply:
yup, I believe so

Reply:
Noice.